wordle-web/app.vue

```diff
@@ -3,44 +3,79 @@
     <v-app>
       <v-app-bar color="primary" :elevation="2">
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-btn icon @click="router.push('/')">
+            <v-icon color="#FFFFFF">mdi-wizard-hat</v-icon> <!-- Wizard hat icon -->
+          </v-btn>
+          <v-btn @click="router.push('/')">
+            Welcome to LexiQuest
```

Comment:
I would make this just "Lexiquest" since it's the title, I like the name/theme though